### ISSUE TITLE
[BUG] - Custom Validation error messages only show persistently in the form when something has been manually typed into the field

### ISSUE THREAD

**Issue by base64000** [formio/formio.js#4084]
**Summary:**
When using Custom Validation expressions and a field is in error state as a result data being loaded into the form, or a calculated value, or a default value (i.e. any mechanism that does not involve direct user interaction with the field), the error label disappears when user interacts with any other field on the form. 

**Why is this a problem**
When using Submit buttons with 'Disable on Form Invalid', the user has no way to determine what field is invalid since the error messages from the custom validations disappear after interaction with any other field on the form

**How to reproduce**
To reproduce, load sandbox id 60a5217f7643cabefc6e03ee

In this sandbox there are three textfields, the first two with a custom validation expression that will result in an error message if the text Sample is present in the textfield. The third textfield has no validations associated to it.

The first textfield is empty by default. The second textfield which has a default value of Sample (to simulate that value being loaded from a service as opposed to being manually entered by the user).

1. When the sandbox is loaded, the second textfield shows the custom validation error
2. Click in the first (empty) textfield and type any character. At this point the error message disappears from the second textfield leaving no hint to the user that the value in that textfield is invalid. 
3. This is very problematic, especially when using Submit buttons marked as 'disable when form is invalid. There is no visual cue to the user to indicate what the error is in the form and they have no immediate way of determining what the problem is

4. Now, manually enter Sample into the first textbox. The field is marked as error, with the text 'Invalid value was entered' showing persistently under the field
5. Click into the third textfield and type any character
6. The custom validation error message stays visible for the first textfield. This is the correct behavior. 
7. No error message is shown in the second textfield even though it is still in an error state - this is incorrect
7. Now click into the second textfield, delete the 'e' from the word Sample and then re-type the 'e'
8. The error message now appears again for the second textfield
9. Click into the third textfield and enter any character
10. The error message for the second textfield is still visible

In conclusion, the error messages related to custom validations are only shown persistently in the form if the user has manually made a change to the field. If the custom validation fails the as part of a calculated value, data being loaded from a service, a default value, or any other mechanism besides direct user interaction with the field, the custom error message disappears as soon as the user interacts with another form element.

The custom error messages should show persistently in the form for all failed validations, not only if the user has manually interacted with the field.

**Comment by Tatsiana8** [comment]
Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
